src/utils/index.js

```diff
@@ -50,3 +51,23 @@ export const invokeOn = (context, fn, ...args) => {
 export const bindTo = (context, ...funcs) => funcs.forEach((func) => {
   if (context[func]) context[func] = context[func].bind(context);
 });
+
+/**
+ * @function isMobileTouchDevice
+ * @description Checks if user client is a mobile device
+ * @returns {boolean}
+ */
+export const isMobileTouchDevice = () => {
```

Comment:
I think a better file for this util would be /src/dom/index.js since everything in here are very general purpose 'pure' functions